gradio-app-highwind-example/requirements.txt

```
@@ -0,0 +1,2 @@
+gradio = "^4.31.4"
+highwind = "1.0.0"
```

@bradleymarques these don't seem to work when running `pip install -r requirements.txt`.

Changing it to this worked for me:
```suggestion
gradio>=4.31.4
highwind==1.0.0
```